specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1106 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureLargeInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/OperationStatusResult"
+            }
+          },
+          "202": {
+            "description": "Accepted. The response indicates that the existing Azure Large Instance is now turning off and includes a 'Location' header to query the operation result.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "URL to track the operation status of the Microsoft Azure Large Instance in the specified location."
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.AzureLargeInstance/operations": {
+      "get": {
+        "tags": [
+          "Common"
+        ],
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "List all management operations supported by the AzureLargeInstance RP": {
+            "$ref": "./examples/AzureLargeInstanceOperations_List.json"
+          }
+        },
+        "summary": "Lists all of the available AzureLargeInstance REST API operations.",
+        "description": "Gets a list of AzureLargeInstance management operations.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
Why null? Keep the pagination enabled for just in case your RP starts supporting many operations.

Reply:
We aim to retain a structure similar to the BareMetal Instance RP whenever feasible in the specs. Also, we don't have plans to add more operations in the near or mid-term future.

Reply:
But adding pagination later is a breaking change for your customers, so better to add now, or never to add if that's the case.

Reply:
After validating our internal behavior, we noticed that we are already supporting that behavior. I've updated the Swagger documentation. Thanks for pointing it out.